### PR TITLE
Detectives gain independency

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

This PR changes the description of the detective's role and shoves them into the civilian category, as they now take answers from not JUST security. While they still have brig access, they are now discouraged from working as a security officer part-time. Instead, they answer requests; illegal or not! Though, if it's a serious case they should still report to security and do the right thing... unless they are a syndicate agent! They can now be a round-start syndicate agent or thief, and are also not mindshielded.

Even as an independent agent, the detective is encouraged to answer to any requests from the security force, but they don’t have to follow their orders. This is similar to the lawyer, as originally the detective had service access and could enter other places if needed. 

The detective also requires 5 hours of service, so they can understand the general department better before working with it, and they spawn with a security-service headset which allows them to listen in to the service department as well.

## Why / Balance

There was discussion brought up about the detective being an independent agent, as seen in this photo:

image

I tried my best to incorporate aspects of this discussion thread, however i am not sure how this would work in regards to the playerbase and the changes it’d bring compared to what people currently understand of this job. At least, i believe this change will allow the detective access to more cases, and perhaps be able to participate in more interesting scenarios.

Please tell me if i worded anything badly, it’s my first time making a change like this and there should be heavy discussion on this topic as it might not be the best change, even if i think it'd be really cool for detectives!

**Changelog**

:cl: Ubaser
- tweak: Detectives now work as independent agents, separate from security, and are encouraged to answer any requests for help.

### CODE REVIEW

**Comment by keronshb** [comment]
Not a review but I'm in favor of this. Previously Det was just just a reflavored Sec role.

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 1]
```diff
@@ -7,12 +7,14 @@
   - !type:DepartmentTimeRequirement
     department: Security
     time: 54000 # 15 hours
+  - !type:DepartmentTimeRequirement
+      department: Civilian
+      time: 18000 # 5 hours
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:12]: i don't get what this is for and it seems strange

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 2]
```diff
@@ -7,12 +7,14 @@
   - !type:DepartmentTimeRequirement
     department: Security
     time: 54000 # 15 hours
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:9]: having sec department time for a non-sec role is kinda strange and it should probably be reduced a bit.

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 3]
```diff
@@ -7,12 +7,14 @@
...
     department: Security
     time: 54000 # 15 hours
+  - !type:DepartmentTimeRequirement
+      department: Civilian
+      time: 18000 # 5 hours
   startingGear: DetectiveGear
   icon: "JobIconDetective"
-  supervisors: job-supervisors-hos
+  supervisors: job-supervisors-security-hire
   canBeAntag: false
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:16]: if it's not a sec role then it probably should be allowed to be an antag and not have mindshield, just like lawyer

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml [hunk 4]
```diff
@@ -7,12 +7,14 @@
...
   access:
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Roles/Jobs/Security/detective.yml:17]: they should probably have extended access for security in case of lowpop.

File: Resources/Locale/en-US/job/job-supervisors.ftl [hunk 5]
```diff
@@ -10,6 +10,7 @@ job-supervisors-service = chefs, botanists, the bartender, and the head of perso
 job-supervisors-engineering = station engineers, atmospheric technicians, and the chief engineer
 job-supervisors-medicine = medical doctors, chemists, and the chief medical officer
 job-supervisors-security = security officers, the warden, and the head of security
+job-supervisors-security-hire = the head of security and whoever hires you
```
Comment (EmoGarbage404) [inline Resources/Locale/en-US/job/job-supervisors.ftl:13]: i would really just cut out the "whoever hires you" since they aren't your supervisors so much as just clientele.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
probably also want to do mapping changes first since there's quite a few maps where det is nestled in security.

**Comment by FairlySadPanda** [comment]
Renaming the role "Private Detective" makes sense if it's a non-Sec role, FWIW.

**Comment by Dutch-VanDerLinde** [comment]
This is great, does the detective still start with armor and their revolver?

**Comment by Nimfar11** [comment]
The idea is interesting. But even if he becomes a detective in his own right, it seems like he should retain a defence against becoming an antag, otherwise it would severely reduce security interactions. Which would end up making the detective cut off, because no one from security or heads wants to work with him, and will just take away his equipment.

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml (hunk 3 above)
Comment (Nimfar11) [reply Resources/Prototypes/Roles/Jobs/Security/detective.yml:16]: This will likely result in cutting out most of the gameplay for the detective.

Comment (Aisu9) [reply Resources/Prototypes/Roles/Jobs/Security/detective.yml:16]: this will also give them a free gun able to shot the 0.45 AP ammo, resulting in 8 TC economy if they ever decide to go loud with their revolver

**Comment by asperger-sind** [comment]
The bartender starts with a shotgun and an armor vest, alike the detective with a revolver and a different armor vest, I don't see bartenders having their roundstart gear confiscated very often (unless its misused), I assume det won't be too different.

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml (hunk 2 above)
Comment (Cyber-Finn) [reply Resources/Prototypes/Roles/Jobs/Security/detective.yml:9]: not necessarily, Detective is still a sec-related role and people need to have played at least some sec to know how to handle situations. For example, you cant just perma/kill someone because they bolted a door. The det needs to know how to handle these various intricate situations, and appropriate sec play-time will allow for this. 15 hours makes sense, since they'd need to have some sec background

File: Resources/Prototypes/Roles/Jobs/Security/detective.yml (hunk 1 above)
Comment (Cyber-Finn) [reply Resources/Prototypes/Roles/Jobs/Security/detective.yml:12]: seems the same as the playtime required for borgs. in this case, you need 5 hours of playtime to take up the role of this specific department.

**Comment by Cyber-Finn** [comment]
Since detectives are now branching out on their own, 
is there the possibility for syndie detectives? or are they still basically as-is?

**Comment by Nimfar11** [comment]
> The bartender starts with a shotgun and an armor vest, alike the detective with a revolver and a different armor vest, I don't see bartenders having their roundstart gear confiscated very often (unless its misused), I assume det won't be too different.

In this case, I was referring to the Evidence Scanner.... or whatever it's called. The detective would be dealing with domestic crimes (which admittedly is pretty rare something interesting), and the sec would keep him away from the crimes that the Syndicate did, doing the work themselves. It seems to me that it will just cut out the core gameplay for the detective.

**Comment by HoidC** [comment]
The detective is meant to investigate Syndicate crimes too, as far as I can tell.

While I understand the reasoning behind this change I feel like it's nerfing a department that is already in a good place balance-wise. The detective is a vital part in identifying traitors and preventing future attacks/hunting down the valuable items that may have been stolen, and to take them away from that by increasing their workload seems like a bad idea for little gain.

**Comment by mirrorcult** [comment]
hadn't considered this but this is a cool concept and im totally willing to try it

**Comment by Cyber-Finn** [comment]
> The detective is meant to investigate Syndicate crimes too, as far as I can tell.
> 
> While I understand the reasoning behind this change I feel like it's nerfing a department that is already in a good place balance-wise. The detective is a vital part in identifying traitors and preventing future attacks/hunting down the valuable items that may have been stolen, and to take them away from that by increasing their workload seems like a bad idea for little gain.

So I defs do agree with you, especially on the "sec is currently balanced" point,
BUT, looking at recent PRs - where they're looking to add Disabler SMGs (Enhanced stun meta), stun/prod sticks, tasers, etc., sec is going to be unbalanced again in the near future,
So perhaps this change is a good thing to pass some difficulty back to sec, without removing/inhibiting ingenuity and content creation

**Comment by K-Dynamic** [comment]
Why not have sec detectives and independent civilian private investigators

**Comment by lzk228** [comment]
> Why not have sec detectives and independent civilian private investigators

too many roles

**Comment by asperger-sind** [comment]
>looking at recent PRs - where they're looking to add Disabler SMGs (Enhanced stun meta), stun/prod sticks, tasers, etc., sec is going to be unbalanced again in the near future

Stunprods are not replacing security gear and won't be used by sec to begin with, while department-related gear is very rare to see being researched, let alone produced; none of the 3 changes actually put sec into a position of odd balance, the point HoidC is making does raise concerns regarding this PR's effect on the state of security.

Why are the changes not making sec unbalanced? The stunprod change is a QoL change changing their sound, size and making them throwable, their stamina damage remains as-is thusly not making them contest batons strength-wise. SMGsablers and tasers being reintroduced are locked behind T2 and T3 arsenal which makes them very rare to see because that requires effort from both sec and sci, the latter may also be unwilling to spend points on that as well.

**Comment by K-Dynamic** [comment]
> > Why not have sec detectives and independent civilian private investigators
> 
> too many roles

Okay fine, at least make it so we can have two or three detectives per map instead of being gimped when one is gibbed in maints

**Comment by BreadyToad** [comment]
What is stopping a sec detective from being a private investigator? As a dec main I can say that the majority of requests I ever got from civilians is "Who stole my cow," "please find my boxing gloves" etc., making these requests almost uninvestigatable and, more importantly, not important to finding out real antags. Detective should be a sec role, because they literally investigate crimes, plus have access to a lethal weapon. Detectives work perfectly as a part of sec, being not a brute force but brains. Additionally, if dec gets removed from sec they will have no choice but to ask for his help anyway, so literally nothing will change apart from dec not being a legal force anymore

**Comment by Tunguso4ka** [comment]
I completely agree with BreadyToad, except for the part for brig access and sec channel (it's already in the PR).

The main problem with removing det's mindshield is that no one in sec and command will trust him.

It's better now (in the sense of security) to just give det's equipment to the secoff every roundstart and let dec solve civ's problems, turning him in the RP-only job.

**Comment by quatre** [comment]
For me the more interesting implications are with a syndicate detective. I don't think it would change much for revs to have them shielded at round-start or not as the det has glasses anyway, and will probably be among the first people to get mindshielded.

About the security records console, we could make it append only, and perhaps (if it has one?) make the server EMAG-able.

**Comment by BreadyToad** [comment]
> For me the more interesting implications are with a syndicate detective. I don't think it would change much for revs to have them shielded at round-start or not as the det has glasses anyway, and will probably be among the first people to get mindshielded.
> 
> About the security records console, we could make it append only, and perhaps (if it has one?) make the server EMAG-able.

Removing shield will basically cut detective from security completely, as it will now be more dangerous than syndie atmos because he has access to a lethal weapon, armor, security channel and IDs and he is the only person who can find other syndies because of his equipment, so if detective will have even a slight chance to be a syndie secoffs will just beat him up and take all his equipment to basically just make one of the officers a detective

**Comment by Cyber-Finn** [comment]
> Removing shield will basically cut detective from security completely, as it will now be more dangerous than syndie atmos because he has access to a lethal weapon, armor, security channel and IDs and he is the only person who can find other syndies because of his equipment, so if detective will have even a slight chance to be a syndie secoffs will just beat him up and take all his equipment to basically just make one of the officers a detective

So I agree with BreadyToad here as well,
It's also gonna make revs a bit difficult, because sec will almost always now push for mindshielding the detective at round-start (Since he is such a massive risk, unlike lawyer), and while they're at it, why not mindshield a few other people?
And as mentioned, secoffs might just bully a detective into handing over their equipment

**Comment by K-Dynamic** [comment]
> > Removing shield will basically cut detective from security completely, as it will now be more dangerous than syndie atmos because he has access to a lethal weapon, armor, security channel and IDs and he is the only person who can find other syndies because of his equipment, so if detective will have even a slight chance to be a syndie secoffs will just beat him up and take all his equipment to basically just make one of the officers a detective
> 
> So I agree with BreadyToad here as well, It's also gonna make revs a bit difficult, because sec will almost always now push for mindshielding the detective at round-start (Since he is such a massive risk, unlike lawyer), and while they're at it, why not mindshield a few other people? And as mentioned, secoffs might just bully a detective into handing over their equipment

Not if sec is reminded that's powergaming and that det's new role is to be respected, like how lawyers aren't automatically mindshielded or treated with extreme prejudice even if they could be revs or syndies

**Comment by BreadyToad** [comment]
> > > Removing shield will basically cut detective from security completely, as it will now be more dangerous than syndie atmos because he has access to a lethal weapon, armor, security channel and IDs and he is the only person who can find other syndies because of his equipment, so if detective will have even a slight chance to be a syndie secoffs will just beat him up and take all his equipment to basically just make one of the officers a detective
> > 
> > 
> > So I agree with BreadyToad here as well, It's also gonna make revs a bit difficult, because sec will almost always now push for mindshielding the detective at round-start (Since he is such a massive risk, unlike lawyer), and while they're at it, why not mindshield a few other people? And as mentioned, secoffs might just bully a detective into handing over their equipment
> 
> Not if sec is reminded that's powergaming and that det's new role is to be respected, like how lawyers aren't automatically mindshielded or treated with extreme prejudice even if they could be revs or syndies

This idea would work on paper, but in reality simply telling sec that it's powergaming won't do much as they still will be extremely cautious of detective and will still try to minimize their relations just to be safe, robbing dec of their gameplay and making him a lonely alcoholic in the bar, because civilian cases are extremely rare, almost always are impossible to solve and are mostly irrelevant to the situation on the station. Moreover, as sec won't have detective's help, antags can work much more aggressively because there is no way to tell who touched stuff or spilled blood due to the detective's scanner being with detective

**Comment by GoodWheatley** [comment]
(essaypost incoming)

While I think this change is fine to try for a few weeks, I have serious doubts about how it's going to work out in the long run. Forensics is a integral part of security being able to catch stealth antags right now, and making the detective not part of security massively disrupts the balance of power.

First off, this is how almost every SS13 server had detective be a role in the past, and all of them (except Goon) ended up changing it to what the role is today. The main complaints were that not being able to trust the detective made it almost impossible to catch antagonists who were being stealthy, and sometimes even ended up with some officers just doing the detective's job. On the other side, detective antagonists are *extremely* strong. They have the ability to give completely false forensics reports and make it impossible for security to catch fellow traitors, and it's similarly extremely hard to detect that a detective is doing this unless they slip up.

Secondly, it's not really clear how making the detective a civilian accomplishes the goal of letting them take on non-sec cases. Nothing was stopping them from doing this in the past, and as a frequent detective player I never turned down someone asking for help; security cases just took priority. It's like saying that if security officers were civilians then people would report more crimes to them. You also can't have them exist in a weird limbo state where they're technically not part of security but have the same antag protection as them, since this creates an administration headache where admins have to constantly check that the detective isn't acting as a full security officer and getting approval to do this since everybody knows they can be trusted. The way we have detective now is in my opinion the most streamlined version, since they have the ability to follow a case completely through and make the arrest on their own, and there's no rules issues with them doing this. 

Finally, forensics being entirely in the hands of a role that can no longer be completely trusted massively disrupts the balance of power against stealth antags. There's four ways a detective and security will generally interact over forensics if this PR is merged:

**Detective is not a antag, Security trusts them**: The way we have detective now. Stealth traitors usually end up being caught, if they didn't cover their tracks well enough.

**Detective is a antag, Security trusts them**: The detective has free reign to lie about everything they scan, meaning they can get innocent people framed with ease. Stealth traitors do not get caught.

**Detective is not a antag, Security does not trust them**: Frustrating for both the detective and security, as the detective has no chance of getting listened to and security has no forensic information to follow up on. Stealth traitors do not get caught.

**Detective is a antag, Security does not trust them**: The *detective* might end up being caught if they mess up, but all the other traitors aren't. Stealth traitors do not end up being caught.

Three out of the four scenarios result in stealth traitors not getting caught and frustration for either security and the detective. As already stated in the second paragraph, simply making it not possible for the detective to roll antag results in security just always treating them as trustworthy, just like atmos techs before they were able to roll traitor.

~~In conclusion~~ That's why I think that this change isn't going to work in the long run. The way detective is now means that forensic evidence comes from a source security can trust, and stealth antags at least have a chance at getting caught.

tldr: If detective is a civilian nobody is going to trust them -> traitors aren't going to be caught unless somebody actually sees them doing traitor stuff

**Comment by IProduceWidgets** [comment]
Just give sec access to the detective scanner and fingerprint pads, and let the secoffs figure it out. I see no reason they can't be optional detectives too if det was already doing it.

**Comment by BreadyToad** [comment]
> Just give sec access to the detective scanner and fingerprint pads, and let the secoffs figure it out. I see no reason they can't be optional detectives too if det was already doing it. Just stick them in the secfab and a couple in the sectech.

Because this would destroy the whole point of a detective job. This is like saying "hey, let's give all engineers atmos tools and atmos access since they are working in the same department". Secoffs and detectives do drastically different jobs which both contribute to the department, with dec being the brains and secoffs being the bruteforce

**Comment by IProduceWidgets** [comment]
> > Just give sec access to the detective scanner and fingerprint pads, and let the secoffs figure it out. I see no reason they can't be optional detectives too if det was already doing it. Just stick them in the secfab and a couple in the sectech.
> 
> Because this would destroy the whole point of a detective job. This is like saying "hey, let's give all engineers atmos tools and atmos access since they are working in the same department". Secoffs and detectives do drastically different jobs which both contribute to the department, with dec being the brains and secoffs being the bruteforce

While I agree, This PR is removing Det from Sec.

**Comment by BreadyToad** [comment]
> While I agree, This PR is removing Det from Sec.

Which is exactly what we shouldn't do, because fixing something that isn't broken is generally a bad idea. If we remove both detective and detective's equipment and make det a possible antag we will make detective's life much, and I mean _much_ harder, because no one from sec will believe him. If we remove detective and his equipment but leave the mindshield, literally nothing will change except his access. If we remove detective and give his equipment to sec - we kill the point of playing as a detective, because civilian cases are very rare and generally very boring, and no one will let you close to real criminal-related cases

**Comment by K-Dynamic** [comment]
> Which is exactly what we shouldn't do, because fixing something that isn't broken is generally a bad idea. If we remove both detective and detective's equipment and make det a possible antag we will make detective's life much, and I mean _much_ harder, because no one from sec will believe him. If we remove detective and his equipment but leave the mindshield, literally nothing will change except his access. If we remove detective and give his equipment to sec - we kill the point of playing as a detective, because civilian cases are very rare and generally very boring, and no one will let you close to real criminal-related cases

Outsourcing them as a civilian role is a double-edged sword. They could check evidence and take workload off sec for civil cases (theft and minor assaults) but be too untrustworthy (despite lawyers and sunglasses existing, as well as possibility of only giving them rubbers).

My bigger concerns however...
There's only ever one detective per station so good luck if: they die, there's mountain of evidence all over the station, or just multiple cases to pursue.

Sec gets tied up in timesinks and are never available for emergencies or patrols; most aggravating when they're all watching a single prisoner be stripped and brigged. In reality, evidence and processing should be best left to detectives and wardens since they'd be the actual brains.

There's basically no system of clues or linking suspects to another; contraband is teleported and used within seconds, glove fibres are too vague (searching engineering lol as though the suspect isn't a greytide), no residue system like gunshots or hair, no CCTV recordings, and station records can be deleted by events now.

Honestly, nobody would notice detectives missing if they were removed in their current state.

**Comment by BreadyToad** [comment]
> Outsourcing them as a civilian role is a double-edged sword. They could check evidence and take workload off sec for civil cases (theft and minor assaults) but be too untrustworthy (despite lawyers and sunglasses existing, as well as possibility of only giving them rubbers).
> 
> My bigger concerns however... There's only ever one detective per station so good luck if: they die, there's mountain of evidence all over the station, or just multiple cases to pursue.
> 
> Sec gets tied up in timesinks and are never available for emergencies or patrols; most aggravating when they're all watching a single prisoner be stripped and brigged. In reality, evidence and processing should be best left to detectives and wardens since they'd be the actual brains.
> 
> There's basically no system of clues or linking suspects to another; contraband is teleported and used within seconds, glove fibres are too vague (searching engineering lol as though the suspect isn't a greytide), no residue system like gunshots or hair, no CCTV recordings, and station records can be deleted by events now.
> 
> Honestly, nobody would notice detectives missing if they were removed in their current state.

This problem is caused by several factors. First of all, fast-paced LRP atmosphere highly impacts the way detectives work. As it is a role which demands actual puzzle-solving skills, many people are either unable to wait untill detective finishes investigation, or unwilling to cooperate and take part in interrogations. On MRP (escpecially where I play), the situation is much better, but here we come to the second issue – undeveloped mechanics. New DNA update helps, but detectives still lack bags for evidence, tape recorders, forensic equipment which can be used in case scanner is stolen/lost, and security records. Overall, detective is much more slower-paced than wizden servers, so they are often forgotten even if they work hard

**Comment by FairlySadPanda** [comment]
As I noted in #contributors I have concerns about this due to my Revs 2.0 refactor I'm proposing, in particular because I'm hoping to be able to give a bit of a reason to exist to the _other_ "has access to the brig but is not really a member of Sec" role.

Specifically, I think these changes are all fine, but removing the Det's mindshield is a change too far at the moment.

**Comment by UbaserB** [comment]
This PR is probably ready to merge now, i edited aspid so it has the detective door and there are no mapping conflicts with these changes now 👍🏻

File: Resources/Prototypes/Roles/Jobs/Civilian/detective.yml [hunk 6]
```diff
@@ -1,38 +1,39 @@
...
+  name: job-name-detective
+  description: job-description-detective
+  playTimeTracker: JobDetective
+  requirements:
+  - !type:DepartmentTimeRequirement
+    department: Security
+    time: 36000 # 10 hours
+  - !type:DepartmentTimeRequirement
+      department: Civilian
+      time: 18000 # 5 hours
```
Comment (metalgearsloth) [inline Resources/Prototypes/Roles/Jobs/Civilian/detective.yml:12]: This needs to be inline.

File: Resources/Prototypes/Entities/Clothing/Ears/headsets.yml [hunk 7]
```diff
@@ -241,6 +241,23 @@
...
+      key_slots:
+      - EncryptionKeySecurity
+      - EncryptionKeyService
+      - EncryptionKeyCommon
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/service.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/service.rsi
+
 - type: entity
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Ears/headsets.yml:261]: Just leave them with the sec headset probably, giving them service is odd.

File: Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml [hunk 8]
```diff
@@ -44,13 +44,12 @@
...
   noSpawn: true
-  parent: ClothingBackpackSatchelSecurity
-  id: ClothingBackpackSatchelSecurityFilledDetective
+  parent: ClothingBackpackSatchel
+  id: ClothingBackpackSatchelFilledDetective
   components:
   - type: StorageFill
     contents:
       - id: BoxSurvivalSecurity
-      - id: Flash
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml:53]: Need migrations for yaml ID changes.

File: Resources/Maps/aspid.yml [hunk 9]
```diff
@@ -49155,6 +49132,31 @@ entities:
...
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
```
Comment (metalgearsloth) [inline Resources/Maps/aspid.yml:49159]: What's goin on here

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Maps/aspid.yml (hunk 9 above)
Comment (UbaserB) [reply Resources/Maps/aspid.yml:49159]: i dunno thats just mapping man a bunch of stuff changes lol

**Comment by Jarmer123** [comment]
My two cents as one of the admins is that this sounds like more work from an administrative side. Now we need to keep security in check to make sure they don't take the detective's forensics scanner and simply ignore them and we need to make sure the detective isn't working as a security officer. And as a player who also likes to play detective/Security, I do not think this change will make detective more fun. Sure, discovering leads is fun but then you don't actually get to follow up on it? "Detective, come scan the cobra for prints" and then he just needs to hand off the printout to sec or he goes back to his office to search up the prints and then tells security on the radio the name of the person (If they aren't an antag of course, and security trusts him in that round). Bready and Goodwheatly make some good points which I agree with and I don't think the general player base will be very happy with these changes. I think security is in a good position right now and I think this could interrupt it quite a lot. At least I think there should be more discussion regarding this before it gets merged.

**Comment by UbaserB** [comment]
I believe there has been enough discussion and what we need more is a test merge, since we dont know how the playerbase will react to this change. Luckily, these changes can be reverted at any time as i have thoroughly ensured it is the case.

**Comment by metalgearsloth** [comment]
Time to get reddit posted.

**Review by metalgearsloth — APPROVED** [review]

**Comment by Clostyandr** [comment]
Ough.... The most worst PR merged ever.

**Comment by metalgearsloth** [comment]
> Ough.... The most worst PR merged ever.

Don't be an asshole on github.

**Comment by Sakamoto4ka** [comment]
Now det its lawyer but not useless

**Comment by FairlySadPanda** [comment]
Is this a test merge or is this planning to stick around?

**Comment by Emisse** [comment]
little bit of this little bit of that well see how it goes

**Comment by metalgearsloth** [comment]
Worst case we get data that it makes the game worse revert and just point to the pr if it ever comes up again.
